libs/features/personalization/entitlements.js

```diff
@@ -0,0 +1,17 @@
+export const ENTITLEMENT_MAP = {
+  '9565ef55-faad-430b-b661-596ba7a036c4': 'all-apps',
+  'c8d50cc2-491e-48df-a1b0-1509f0ca7323': 'photoshop',
```

Comment:
Can we generate this list for all the available products already?

Reply:
I'm still waiting for the official list, these were the test values I was given.  Will use a followup PR once we get the values.